Makefile

```diff
@@ -34,12 +36,26 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 .PHONY: lint
-lint: ## Run golangci-lint over the codebase.
-	${GOLANGCI_LINT} run ./... --timeout 5m -v ${GOLANGCI_LINT_EXTRA_ARGS}
+lint: golangci-lint modernize
 
 .PHONY: lint-fix
-lint-fix: GOLANGCI_LINT_EXTRA_ARGS := --fix
-lint-fix: lint ## Run golangci-lint over the codebase and run auto-fixers if supported by the linter
+lint-fix: golangci-lint-fix modernize-fix
+
+.PHONY: golangci-lint
+golangci-lint: ## Run golangci-lint over the codebase.
+	${GOLANGCI_LINT} run ./... --timeout 5m -v ${GOLANGCI_LINT_EXTRA_ARGS}
+
+.PHONY: golangci-lint-fix
+golangci-lint-fix: GOLANGCI_LINT_EXTRA_ARGS := --fix
+golangci-lint-fix: golangci-lint ## Run golangci-lint over the codebase and run auto-fixers if supported by the linter
+
+.PHONY: modernize
+modernize: ## Run modernize on the codebase.
+	go run golang.org/x/tools/gopls/internal/analysis/modernize/cmd/modernize@v${MODERNIZE_VERSION} -diff ./...
```

Comment:
Does it not need to be added to the tools module? Like the tool directive for `golangci-lint`?